projects/igniteui-angular/src/lib/checkbox/checkbox.component.ts

```diff
@@ -278,22 +286,37 @@ export class IgxCheckboxComponent implements ControlValueAccessor, EditorProvide
      */
     @HostBinding('class.igx-checkbox--checked')
     @Input()
-    public checked = false;
+    public get checked() {
+        return this._checked;
+    }
+    public set checked(value: boolean) {
```

Comment:
@damyanpetev I'm going to leave it as is, because the checked input it meant to be used to bind to a model property, which defeats the purpose of supporting it the way we do `required`.